doc/index.rst

```diff
@@ -162,7 +162,7 @@ PennyLane is **free** and **open source**, released under the Apache License, Ve
    :hidden:
 
    code/init
-   code/decorator
+   Creating QNodes <code/decorator>
```

Comment:
This is still out of place in the rendered docs. "Creating QNodes" is the only heading that has a subheading ("The QNode decorator") that just links to the same place/page.

Reply:
Could we title the overall page heading "Creating QNodes", a couple sentences/paragraph about what a QNode is, and how it's the core piece of PL, then have a subheading "The QNode decorator", followed by the current text?